bug(source dist): Error when building source dist

```
  File "setup.py", line 38
    def build_extension(self, ext: Extension) -> None:
                                 ^
SyntaxError: invalid syntax
Error: Process completed with exit code 1.
```

The release workflow fails only on macOS https://github.com/tensorchord/envd/runs/6912751791?check_suite_focus=truehttps://github.com/tensorchord/envd/runs/6912751791?check_suite_focus=true

Do you have any clue about it? @kemingy

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -78,7 +78,7 @@ jobs:
         CIBW_ARCHS: auto64
     - name: Build source distribution
       run: |
-        python setup.py sdist
+        python3 setup.py sdist
         mv dist/*.tar.gz wheelhouse/
     - name: Upload to PyPI    
       env:

